README.md

```diff
@@ -6,5 +6,17 @@ Usage is simple, on any request to `/` the request will result in a `200` respon
 
 A Docker image is available at: `quay.io/brancz/prometheus-example-app:v0.2.0`
 
+## Deploying in a Kubernetes cluster
+
+First, deploy one instance of this example application, which listens and exposes metrics on port 8080 using the following [Deployment](manifests/deployment.yaml) configuration.
+
+Then, in order for Prometheus to be able to scrape these metrics, you will need to deploy a Prometheus Operator. To do so, you can follow this guide: [quickstart][prometheus-operator-quickstart].
```

Comment:
I like to avoid you in technical docs. So maybe something like:
```suggestion
Then, in order for Prometheus to be able to discover and scrape these metrics, in this repository we provide PodMonitor as an example, for that Prometheus Operator needs to be running the cluster. To do so, you can follow this guide: [quickstart][prometheus-operator-quickstart].
```